ed25519jni/rust/Cargo.toml

```diff
@@ -7,11 +7,21 @@ publish = false
 edition = "2018"
 
 [dependencies]
-ed25519-zebra = { path = "../../", version = "3.0.0" }
+der = { version = "0.7.1", optional = true }
+ed25519-zebra = { path = "../../", version = "3.1.0", features = ["pem", "pkcs8"] }
 failure = "0.1.8"
-jni = "0.18.0"
+jni = "0.21.1"
+pkcs8 = { version = "0.10.1", optional = true, features = ["alloc", "pem"] }
+ed25519 = { version = "2.2.0", features = ["alloc", "pem"] }
 
 [lib]
 name = "ed25519jni"
 path = "src/lib.rs"
 crate-type = ["staticlib", "cdylib"]
+
+[features]
+nightly = []
+default = ["std"]
+pem = ["der"]
+pkcs8 = ["dep:pkcs8"]
```

Comment:
I'd prefer consistency here, just in case. I don't think the linking is really an issue unless you're running the tests, and the JNI README has directions on how to properly run the tests.